apps/src/lab2/types.ts

```diff
@@ -405,3 +405,12 @@ export interface LabProps<
   levelProperties: T;
   initialSources?: U;
 }
+
+export enum Tabs {
```

Comment:
Can we put this in a `ResourcePanel/types` or similar as they only apply to resource panel and related components?